lib/elixir/lib/supervisor.ex

```diff
@@ -97,7 +97,7 @@ defmodule Supervisor do
   Supervisors support different strategies; in the example above, we
   have chosen `:one_for_one`. Furthermore, each supervisor can have many
   workers and supervisors as children, each of them with their specific
-  configuration, shutdown values, and restart strategies.
+  configuration, shutdown values, and supervision strategies.
```

Comment:
In this case, however, I believe we were referring to the restart value, since we are talking about the children. Maybe we can use "shutdown and restart values"?

Reply:
You're right, of course :). Since this seems to be discussing child specs, maybe this sentence could be changed to

> Furthermore, each supervisor can have many
   workers and/or supervisors as children, with each one having its own configuration (as outlined in the “Child specification” section).

Reply:
Sounds good to me!

Reply:
Updated, as discussed